### ISSUE TITLE
Origin HTML/XML becomes overwritten if a translation is created in EN(US)

### ISSUE THREAD

**Issue by rk-aix** [odoo/odoo#162763]
Impacted versions: 16 & 17

Steps to reproduce:

Scenario: Default language DE, additional languages FR, NL, EN etc. installed.

In Website Editor start to create a page in DE.
HTML/XML is created correctly in DE with all codes, scripts, tags etc.
You are able to edit HTML/XML in Editor and all is going fine.
After you are satisfied with your page, in next step you create the translations in any language you want, all is going fine... The HTML/XML is still the origin that you've created.
Now in final step you perhaps want to create the translation for EN(US), do it and your developed HTML/XML becomes directly overwritten by a variant from the EN(US) translation.
From now on all is going wrong, the translations become unstable/unlogical/inconsistant, changes in HTML/XML will directly affect all other translations up to trace backs from Odoo.

To check this behaviour it is enough to create only a head-line with a test text in Website-Editor, no need to create a full page...

Also if you just (want to) change some little code in your default language all translations become affected, still if there were previous translations stored.

The affect does not appear if you decide to translate to EN(UK,CA...)

This remembers me to https://github.com/odoo/odoo/issues/124124 and https://github.com/odoo/odoo/issues/150829

Perhaps it's only a small workout to correct, otherwise we're building on sand...
This behaviour i recognized after a client decided to expand his (non-english) web-site from one to some more different languages. Therefore - Be aware - Please don't try this in your production or own development environment, use RunBot or a blank database and you're save ;-)

Current behavior: overwriting origin HTML/XML after translation to en_US

Expected behavior: NOT overwriting origin HTML/XML after translation to en_US

**Comment by 88kbbq** [comment]
Yes, I just went through some Chinese products to translation to English, but all Chinese product names were overwritten. This bug right here tells me Odoo is not mature enough for our company. Will stop implementation.
